inventories/group_vars/all

```diff
@@ -86,8 +86,8 @@ amqp:
     vhost_encoded:
     mgmt_port:
 
-amqp_de_uri: "{{amqp.de.scheme}}://{{amqp.de.user}}:{{amqp.de.password}}@{{amqp.de.host}}:{{amqp.de.port}}/{{amqp.de.vhost_encoded}}"
-amqp_irods_uri: "{{amqp.irods.scheme}}://{{amqp.irods.user}}:{{amqp.irods.password}}@{{amqp.irods.host}}:{{amqp.irods.port}}/{{amqp.irods.vhost_encoded}}"
+amqp_de_uri: "{{amqp.de.scheme}}://{{amqp.de.user}}:{{amqp.de.password|urlencode}}@{{amqp.de.host}}:{{amqp.de.port}}/{{amqp.de.vhost_encoded}}"
+amqp_irods_uri: "{{amqp.irods.scheme}}://{{amqp.irods.user}}:{{amqp.irods.password|urlencode}}@{{amqp.irods.host}}:{{amqp.irods.port}}/{{amqp.irods.vhost_encoded}}"
```

Comment:
I like this. Outside of scope here, but probably it's how we should be doing the thing that's currently done with `{{amqp.vhost_encoded}}` too (which is just manual URL encoding).